entrypoint/rancher/rancher_suite_test.go

```diff
@@ -0,0 +1,57 @@
+package rancher
+
+import (
+	"flag"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/rancher/distros-test-framework/factory"
+	"github.com/rancher/distros-test-framework/pkg/customflag"
+	"github.com/rancher/distros-test-framework/pkg/testcase"
+	"github.com/rancher/distros-test-framework/shared"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+func TestMain(m *testing.M) {
+	var err error
+	flag.Var(&customflag.ServiceFlag.ClusterConfig.Destroy, "destroy", "Destroy cluster after test")
+	flag.StringVar(&customflag.ServiceFlag.ExternalFlag.CertManagerVersion, "certManagerVersion", "v1.11.0", "cert-manager version that will be deployed on the cluster")
+	flag.StringVar(&customflag.ServiceFlag.ExternalFlag.RancherHelmVersion, "rancherHelmVersion", "v2.8.0", "rancher helm chart version to use to deploy rancher manager")
+	flag.StringVar(&customflag.ServiceFlag.ExternalFlag.RancherImageVersion, "rancherImageVersion", "v2.8.0", "rancher version that will be deployed on the cluster")
+	flag.Parse()
+
+	_, err = shared.EnvConfig()
+	if err != nil {
+		return
+	}
+
+	os.Exit(m.Run())
+}
+
+func TestRancherSuite(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Create Cluster Test Suite")
+}
+
+var _ = BeforeSuite(func() {
+	testcase.TestConfigVarVal(GinkgoT(), "create_lb", "true")
```

Comment:
those validation should be better living in the main

because there u might already have access to some env + u can already access the product by
```
_, err = shared.EnvConfig()
if err != nil {
return
}

to


cfg, err = shared.EnvConfig()
if err != nil {
return
}

cfg.Prod.
```